src/flash/handlers/SwEthHandler.sol

```diff
@@ -29,12 +41,24 @@ contract SwEthHandler is UniswapFlashswapHandler, BalancerFlashloanDirectMintHan
         UniswapFlashswapHandler(_swEthPool, true)
     { }
 
-    function _depositWethForLst(uint256 wethAmount) internal override returns (uint256) {
-        WETH.withdraw(wethAmount);
-        return ISwEth(address(LST_TOKEN)).depositForLst(wethAmount);
+    /**
+     * @notice Unwraps weth into eth and deposits into lst contract.
```

Comment:
How about WstEth, SwEth, EthX, ETH, and WETH?

Reply:
Isn't this a mix of camel case and normal ticker symbols? The camel cases with capitalization is only used for naming contracts. The idea is to just follow one so there's no confusion. 

If we don't want to do lowercase 'eth' and 'weth' since that conflicts with some contract variables, how about using normal ticker symbols swETH, wstETH, ETHx, ETH, WETH for all comments or informational things, and using camelcases only when referring to specific variables?